packages/gatsby-1-config-css-modules/__tests__/index.js

```diff
@@ -0,0 +1,38 @@
+var _require = require(`../index`),
```

Comment:
Looks like this was accidentally committed?

Reply:
yeah, a failure to handle the right babel ignores i think